### DISCUSSION TITLE
Crust MainNet Launch Checklist

### DISCUSSION THREAD

**Discussion by badkk** [crustio/crust#693] [Announcements]
- [x] Technical and economic parameters of Crust mainnet are finalized
- [x] Mainnet code is complete
  - [x] [Chain code](https://github.com/crustio/crust/tree/mainnet-staging) is complete
  - [x] [Chain code](https://github.com/crustio/crust/tree/mainnet-staging) testing is complete
  - [x] [Storage Worker](https://github.com/crustio/crust-sworker/tree/mainnet-staging) module code is complete
  - [x] [Storage Worker](https://github.com/crustio/crust-sworker/tree/mainnet-staging) module code testing is complete
- [x] [CRU claim](https://wiki.crust.network/docs/en/claims) on Crust mainnet is enabled
- [x] Auditing for Crust mainnet code is complete
- [x] Mainnet [Storage Manager](https://github.com/crustio/crust-smanager) technical test complete
- [x] The mainnet is open and all nodes are allowed to join
- [x] DSM (Decentralized Storage Market) test completed
- [x] DSM (Decentralized Storage Market) features are enabled

**Comment by Akitoriowo** [comment]
Crust 🚀🚀🚀

**Reply by Destore2023** [reply]
great to hear this!

**Reply by teaner** [reply]
Crust 🚀🚀🚀

**Reply by LowEntropyBody** [reply]
🚀🚀🚀

**Comment by teaner** [comment]
Crust 🚀🚀🚀

**Comment by CallmeOrange** [comment]
congratulations Crust！！
